draft-gruessing-moq-requirements.md

```diff
@@ -206,15 +207,25 @@ Our goal in this section is to understand the requirements that result from the
 
 ## Specific Protocol Considerations {#proto-cons}
 
-### Common push protocol for ingest and distribution
+In order to support the various topologies and patterns of media flows with the protocol, the protocol MUST support both sending and receiving of media streams, as separate actions or concurrently in a given connection.
```

Comment:
I'm only mentioning this here, because it's the first occurence of BCP 14 language in the PR, but it's worth mentioning that #75 is still open, awaiting discussion with the working group.

Reply:
I was kind of hoping that actually having some prose to show just how normative language in the document could work might aid the discussion.  If for some reason the WG decides that it's unacceptable, it would take a bit of `s/SHOULD/should/g` in a future PR to address. I'm prepared to do that reluctantly if that's the case.

Reply:
Hi, @fiestajetsam, I'm fine with that strategy. My impression is that (much like other people who served on the IESG at the same time with him) @fluffy has clear memories of all the ways that BCP 14 language in requirements documents can go sideways (he was on the IESG before me, so I don't know all the details, but I'm remembering those problems being equal portions of "you can't adopt that approach, because it violates a MUST requirement" and "you can't adopt that approach, because there's no requirement that requires it". 

He might also be scarred by other experiences, of course. :yum:

But we've got an issue for that, and we can talk about it after requesting adoption.